examples/java/README.md

```diff
@@ -0,0 +1,18 @@
+## Run
+
+Ensure that you have an instance of ValKey running on "localhost" on "6379". Otherwise, update glide.examples.ExamplesApp with a configuration that matches your server settings.
```

Comment:
nit: Valkey is spelled with a lowercase "k" on the official website.
```suggestion
Ensure that you have an instance of Valkey running on "localhost" on "6379". Otherwise, update glide.examples.ExamplesApp with a configuration that matches your server settings.
```